crates/swc_ecma_transforms_module/tests/fixture/common/ignore-dynamic/1/.input.js

```diff
@@ -0,0 +1,11 @@
+import foo from "foo";
+
+function foo() {
+    await import("foo");
+
+    callback(() => import("foo"));
+}
+
+import("side-effect");
+
+await import("awaited");
```

Comment:
I got an error
```
top level await requires target to es2017 or higher and topLevelAwait:true for ecmascript
```

How to set the parser target with `test_fixture`? @kdy1

Reply:
You can change https://github.com/swc-project/swc/blob/29522b60a53e3c3dcd7078cc7abd4b9615ee6461/crates/swc_ecma_transforms_testing/src/lib.rs#L163 to use https://docs.rs/swc_ecma_parser/latest/swc_ecma_parser/struct.Parser.html#method.new_from with `EsVersion::latest()`.


(This is my mistake)